packages/wonder-blocks-dropdown/src/util/constants.ts

```diff
@@ -48,6 +48,7 @@ export const defaultLabels = {
 } as const;
 
 export const defaultComboboxLabels: ComboboxLabels = {
+    clearSelection: "Clear selection",
```

Comment:
praise: I like that this copy could be used for both single and multiple selection types!